sympy/interactive/printing.py

```diff
@@ -321,8 +337,10 @@ def init_printing(pretty_print=True, order=None, use_unicode=None,
     euler: boolean, optional, default=False
         Loads the euler package in the LaTeX preamble for handwritten style
         fonts (http://www.ctan.org/pkg/euler).
-    forecolor: string, optional, default='Black'
-        DVI setting for foreground color.
+    forecolor: string, optional, default='Auto'
+        DVI setting for foreground color. 'Auto' means that either 'Black' or
+        'White' will be selected based on a guess of the IPython terminal color
+        setting.
```

Comment:
I will update the description with the mentioning of gray and a bit about how this is decided (maybe a link to the IPython description of the colors variable).